starter-code/js/index.js

```diff
@@ -42,11 +61,16 @@ window.onload = function(){
   var calculatePriceButton = document.getElementById('calc-prices-button');
   var createItemButton = document.getElementById('new-item-create');
   var deleteButtons = document.getElementsByClassName('btn-delete');
+  var inputs = document.querySelectorAll(".number-Items").value;
```

Comment:
You don't have any class named .number-Items in your HTML.

document.querySelectorAll(".number-Items") returns an array. You can't get a .value of an array.